lib/appsignal/probes/erlang_probe.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Appsignal.Probes.ErlangProbe do
+  def call do
+    io_metrics()
+    scheduler_metrics()
+    process_metrics()
+    memory_metrics()
+  end
+
+  defp io_metrics do
+    {{:input, input}, {:output, output}} = :erlang.statistics(:io)
+    Appsignal.set_gauge("erlang_io", Kernel.div(input, 1024), %{kind: "input"})
```

Comment:
Can we rename these tag keys to `type`?
```suggestion
    Appsignal.set_gauge("erlang_io", Kernel.div(input, 1024), %{type: "input"})
```

Context, also here: https://github.com/appsignal/appsignal-ruby/pull/488#discussion_r265551406